packages/mobile/src/redux/migrations.ts

```diff
@@ -344,4 +344,5 @@ export const migrations = {
     },
   }),
   23: (state: any) => state,
+  24: (state: any) => state,
```

Comment:
wondering if you want to do a `invites: undefined` in the state? i guess it is not necessary but would be nice to have it here or in the test schemas file to see what changed?